coc.md

```diff
@@ -0,0 +1,106 @@
+Except in the most egregious cases, organizers and moderators will follow these guidelines:
+
+- First, attempt to privately message or speak with the person and point out to them that their behavior violates the CoC. Include a link or try to be specific about what area of the the CoC has been violated.
+- If the person does not respond to a request or is not willing to talk, escalate to the private #gopher-admins channel on gophers.slack.com or assemble other team members to discuss the incident.
+- Once the issue has been discussed, follow the reporting guidelines below.
+
+In those cases of obvious, flagrant violations in our online communities, moderators should remove the post immediately and bring the issue to the attention of the private gophers-admin slack channel.
+
+#### Dealing with Spam
+Although not explicitly called out in this CoC, we ask that all moderators be aware of and intervene in the case of spam.
+
+Spam is a post or a series of posts where the content is primarily solicitation, nonsense, overly short/long, or off-topic. Posts where the content is not hand-crafted or is too generic/useless may also be considered spam. Spam accounts are user accounts which post spam content or are created in series to secure access to the site and its privileges, or both. Spam posts and accounts may be automated or manual. If a moderator notices multiple spam accounts in succession, the IP address may be blocked.
+
+We realize spam can be hard to define in some cases, so we will rely on the community or moderators to identify and flag it and handle it appropriately. Obvious spam posts should be promptly deleted; non-obvious spam may be flagged for review. Users are encouraged to flag posts as spam to help bring them to a moderator’s attention.
+
+Not all spam posts are by spam users. For example, a legitimate user account may be compromised. If a moderator notices a reputable user account posting spam, the spam post(s) should be deleted immediately and the user notified. If spam posts continue, the account may be suspended until the user can regain control of his/her account.
+
+Commercial content is allowed if relevant to answering a question or keeping inline with a topic.
+
+#### Reporting guidelines
+If someone is subject to or witnesses unacceptable behavior, or has any other concerns, they are strongly encouraged to notify an organizer, moderator or a GB Leadership team member as soon as possible. Every reported concern will follow this workflow until resolved:
+
+- Make contact by directly talking to an organizer, moderator or emailing support@gobridge.org
+  - Your email message will reach the **GB Leadership team** only.
+  - Reports will be confidential within the organizers team, moderators team or GB’s Leadership team.
+
+- You will receive a response within 24-48 hours (very likely sooner) in our online communities. For our offline communities and events, a response should be expected within an hour or two.
```

Comment:
I suggest removing this bullet point as we don't have enough people to give a strong timeline.

Reply:
I'm not sure if you are keeping this because you wanted to or you missed my comment? I am concerned that we can't meet the SLA listed here.

Reply:
> I'm not sure if you are keeping this because you wanted to or you missed my comment? I am concerned that we can't meet the SLA listed here.

Oh, I am sorry I missed this comment. I'll get it sorted. Thanks for pointing out :)